token-metadata/program/src/processor/burn/nonfungible_edition.rs

```diff
@@ -0,0 +1,162 @@
+use crate::state::{MasterEdition, MasterEditionV2, EDITION_MARKER_BIT_SIZE};
+
+use super::*;
+
+pub(crate) fn burn_nonfungible_edition(ctx: &Context<Burn>) -> ProgramResult {
+    let edition_info = ctx.accounts.edition_info.unwrap();
+
+    let parent_mint_info = ctx
```

Comment:
If we are changing the `parent_*` naming, the errors need to be updated as well.